Users/router.js

```diff
@@ -1,15 +1,20 @@
 var express = require('express');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:80085');
+var userModels = require('./userModels');
+var StudentModel = userModels.Student;
+var AdminModel = userModels.Admin;
 var userService = require('./service.js');
 var userRouter = express.Router();
 
 // Create User
-userRouter.post('/register', (req, res) => {
+userRouter.post('/:uid', (req, res) => {
```

Comment:
The URI for this should just be ```'/'```, since you wouldn't know the UID of a user that hasn't been created, yet.